index.html

```diff
@@ -9,222 +9,249 @@
+            </p>
+            <ul class="styling3">
+                <li>css</li>
+                <li><img src="Rectangle.70.svg" alt="border"></li>
+                <li>html</li>
+                <li><img src="Rectangle.70.svg" alt="border"></li>
+                <li>bootstrap</li>
+                <li><img src="Rectangle.70.svg" alt="border"></li>
+                <li>Ruby</li>
+
+
+            </ul>
+
+            <button type="button" class="button1">See project</button>
+
+
+        </div>
+
+
+        <div class="redifining">
+            <img class="picture" src="Img.Placeholder.svg" alt="picture">
+            <h2 class="text">Multi-Post Stories</h2>
+            <p class="paragraph-text">A daily selection of privately personalized reads; no accounts or sign-ups
+                required. has been the
+                industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy
+                text.
+            </p>
+            <ul class="styling3">
+                <li>css</li>
+                <li><img src="Rectangle.70.svg" alt="border"></li>
+                <li>html</li>
+                <li><img src="Rectangle.70.svg" alt="border"></li>
+                <li>bootstrap</li>
+                <li><img src="Rectangle.70.svg" alt="border"></li>
+                <li>Ruby</li>
+
+
+            </ul>
+
+            <button type="button" class="button1">See project</button>
+
+
+        </div>
+    </section>
+    <h1 class="maintextfooter">Contact us</h1>
+    <p class="paragraph-text1">If you have an application you are interested in developing, a feature that you need built or a project that
+        needs coding. I’d love to help with it </p>
+    <footer class="third width">
+        <form action="https://formspree.io/f/mgeddrqw"
+        method="POST">
+            <input type="text" name="name" id="name" placeholder="YourName" class="input123" maxlength="30" required=""><br>
+            <input type="email" name="email" id="email" placeholder="youremail@gmail.com" class="input1234" required="" ><br>
+            <textarea name="Message" id="Message" cols="30" rows="10" placeholder="Leave your message" class="input12345" maxlength="500" required=""></textarea>
+            <button class="button12">See project</button>
+        </form>
+        <img src="Rectangle.421.svg" alt="separator" class="separator-text">
+
+        <div class="icon">
+
+            <ul class="styling1">
+                <li class="styling1"><img src="Vector.1.svg" alt="icon"></li>
+                <li class="styling1"><img src="Group.png" alt="icon"></li>
+                <li class="styling1"><img src="Vector.3.svg" alt="icon"></li>
+                <li class="styling1"><img src="Vector.icon.svg" alt="icon"></li>
+                <li class="styling1"><img src="Vector.5.svg" alt="icon"></li>
+    
+            </ul>
+        </div>
+        <p class="text-footer">jessemccallums2001@gmail.com</p>
+    </footer>
```

Comment:
Your design doesn't look quite like the template design.

Here's your design:

<img width="243" alt="Screen Shot 2022-05-31 at 8 39 16" src="https://user-images.githubusercontent.com/82426682/171187266-5f4d5bf6-831d-4de3-beff-422c537e5370.png">

Here's the template design:

<img width="322" alt="Screen Shot 2022-05-31 at 8 40 07" src="https://user-images.githubusercontent.com/82426682/171187449-47e3d200-51c4-44a7-a4b2-8e39fda9e244.png">

Do you see how different margins, padding, heights, and the overall layout are? Kindly fix that.

Additionally, there's a strange horizontal scrolling behavior you should fix as well.

Reply:
That's weird, here in my browser looks like this:

![image](https://user-images.githubusercontent.com/101757754/171190426-a00ad27d-7dfd-4cb3-9f4a-6e08d4a7056d.png)

Reply:
Hi Sir, I already solved it, check the next pull request